sales_team_security/models/res_users.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    section_ids = fields.Many2many(
+        comodel_name="crm.case.section", string="Sales teams",
+        relation='sale_member_rel', column1='member_id', column2='section_id')
```

Comment:
No, we can't change this, as the table already exists, and have these column names.